### PR TITLE
Disable DependentUpon convention for EntityFramework.csproj

### PR DESCRIPTION
**Pull request by rainersigwald**
Code generated by Resources.tt->Resources.cs assumes a resource naming convention that isn't compatible with the DependentUpon convention, so disable it for this project.

### CODE REVIEW

**Comment by wtgodbe** [comment]
Still seeing the test failures:

> System.Resources.MissingManifestResourceException : Could not find the resource \"System.Data.Entity.Properties.Resources.resources\" among the resources \"System.Data.Entity.Resources.Strings.resources\", \"System.Data.Resources.AnnotationSchema.xsd\", \"System.Data.Resources.CodeGenerationSchema.xsd\", \"System.Data.Resources.CSDLSchema_1.xsd\", \"System.Data.Resources.CSDLSchema_1_1.xsd\", \"System.Data.Resources.CSDLSchema_2.xsd\", \"System.Data.Resources.CSDLSchema_3.xsd\", \"System.Data.Resources.EntityStoreSchemaGenerator.xsd\", \"System.Data.Resources.SSDLSchema.xsd\", \"System.Data.Resources.SSDLSchema_2.xsd\", ... embedded in the assembly \"EntityFramework\", nor among the resources in any satellite assemblies for the specified culture. Perhaps the resources were embedded with an incorrect name.

File: src/EntityFramework/EntityFramework.csproj [hunk 1]
```diff
@@ -7,6 +7,9 @@
     <Description>EntityFramework.dll</Description>
     <AssemblyVersion>6.0.0.0</AssemblyVersion>
     <DefineConstants>$(DefineConstants);INTERNALS_INVISIBLE</DefineConstants>
+
+    <!-- Properties\Resources.tt expects a path that does not match the class in Properties\Resources.cs -->
```
Comment (nguerrera) [inline src/EntityFramework/EntityFramework.csproj]: Is it worth changing the code to use the conventional name?

Comment (rainersigwald) [reply src/EntityFramework/EntityFramework.csproj]: This is a great question I leave up to the maintainers. It didn't seem trivial, but I don't know the history of the `Resources.tt` approach or the details of its implementation.

**Comment by rainersigwald** [comment]
So it looks like I fixed _almost_ all the errors, but there's two instances of this:

```
Actual string does not match the message defined in resources.
Expected:'Spatial readers can only be produced from readers of type SqlDataReader. A reader of type Castle.Proxies.DbDataReaderProxy was provided.',
Actual:'The provider did not return a 'DbSpatialServices' instance. In order to use the 'DbGeography' or 'DbGeometry' spatial types the EF provider being used must support spatial types and all prerequisites for the provider must be installed. See http://go.microsoft.com/fwlink/?LinkId=287183 for details.'
Expected: True
Actual: False
```

@wtgodbe do you think that's resource-related?

**Comment by rainersigwald** [comment]
@ajcvickers, do you know (or know who might know) what might be up with that remaining error? Is it likely to be related to resource naming?

**Comment by ajcvickers** [comment]
@bricelam @rainersigwald @dougbu This was happening while I was on vacation; sorry for missing it. Does this change need to go in for the 3.1 release train?

**Comment by dougbu** [comment]
I believe @wtgodbe incorporated this proposal in #1291. @rainersigwald / @wtgodbe do we need this PR for something else?

**Comment by ajcvickers** [comment]
This PR has been closed because EF6 is no longer being actively developed. We are instead focusing on stability of the codebase, which means we will only make changes to address security issues. See the repo [README](https://github.com/dotnet/ef6/blob/main/README.md) for more information.

**Pull request closed without merging**